frontend/src/components/feature/chat/ChatMessage/Renderers/FileMessage.tsx

```diff
@@ -12,6 +12,11 @@ import { memo } from "react"
 import { toast } from "sonner"
 import { useIsDesktop } from "@/hooks/useMediaQuery"
 
+import path from 'path';
+import { fromPath } from 'pdf2pic';
+import fs from 'fs';
+
+import test from '../../../../../../../../../../../../Downloads/test.png'
```

Comment:
Remove the hardcoded test image import and replace it with a more appropriate placeholder or loading state.